src/main/java/datastax/cdm/job/ExceptionHandler.java

```diff
@@ -0,0 +1,32 @@
+package datastax.cdm.job;
```

Comment:
#nitpick not related to this PR, but we should standardize on the package naming convention. Maybe `com.datastax.cdm.job`? 🤔

Reply:
Sounds good. Let us confirm in our Thu call.